src/applications/vaos/tests/e2e/page-objects/DateTimeSelectPageObject.js

```diff
@@ -26,10 +26,10 @@ export class DateTimeSelectPageObject extends PageObject {
 
   assertUrl() {
```

Comment:
This change is to overcome an axe check error for color contrast. @vbahinwillit supplied the code.